### PR TITLE
NT-849 Added Expand/Collapse side menu

### PR DESCRIPTION
**Pull request by slomianadam**

### CODE REVIEW

File: src/components/side-menu-web/side-menu-web.scss [hunk 1]
```diff
@@ -36,6 +37,26 @@ side-menu-web {
...
+        .side-menu-web{
+            * {
+                display: none;
+            }
+            .menu-logo-wrapper, .menu-logo-wrapper * {
+                margin-left: -0.75rem;
+                padding: 0px;
+                height: 45px;
+                width: 18rem;
+                display: block;
```
Comment (reg0) [inline src/components/side-menu-web/side-menu-web.scss]: sorry, my mistake, this is wrong here:
only ```display: block;``` should be applied to ```.menu-logo-wrapper * ```
rest should be only for ```.menu-logo-wrapper```

File: src/components/side-menu-web/side-menu-web.html [hunk 2]
```diff
@@ -1,9 +1,11 @@
 
-    <div class="side-menu-web">
-      <img class="menu-logo" src="/assets/images/Worktime_logo_black.svg"/>
-      <page-list [forWeb]="true"></page-list>
-      <user-profile [isOnLightBackground]="true"></user-profile>
-      <logout-button [isOnLightBackground]="true"></logout-button>
+    <div  class="side-menu-web">
+      <div class="menu-logo-wrapper">
```
Comment (reg0) [inline src/components/side-menu-web/side-menu-web.html]: please fix indentation in following 3 lines

File: src/components/side-menu-web/side-menu-web.ts [hunk 3]
```diff
@@ -22,4 +28,13 @@ export class SideMenuWebComponent {
         this.menuCtrl.close();
     }};
 
+    hideSideMenu(){
```
Comment (reg0) [inline src/components/side-menu-web/side-menu-web.ts]: this method could be named ```toggleSideMenu```

File: src/components/side-menu-web/side-menu-web.ts [hunk 4]
```diff
@@ -22,4 +28,13 @@ export class SideMenuWebComponent {
...
+      this.sideMenuHidden = !this.sideMenuHidden;
+      let element = document.getElementsByClassName("worktime-menu")[0];
+      if (this.sideMenuHidden) {
```
Comment (reg0) [inline src/components/side-menu-web/side-menu-web.ts]: maybe this field is not needed anymore?
it'd be enough to check if that class is already there

File: src/components/side-menu-web/side-menu-web.ts [hunk 5]
```diff
@@ -22,4 +28,13 @@ export class SideMenuWebComponent {
...
+        element.className += " collapsed";
```
Comment (NikolaiLarsen) [inline src/components/side-menu-web/side-menu-web.ts]: you could maybe use ngClass for this instead :)

Comment (reg0) [reply src/components/side-menu-web/side-menu-web.ts]: it's not that easy, because it has to be set on an element outside of this component

Comment (NikolaiLarsen) [reply src/components/side-menu-web/side-menu-web.ts]: oh right, i didnt notice that

Comment (NikolaiLarsen) [reply src/components/side-menu-web/side-menu-web.ts]: Then we could maybe have a sideMenuOpen field in redux and use that for ngClass?

Comment (reg0) [reply src/components/side-menu-web/side-menu-web.ts]: @azslomian maybe it'd be cleaner to use Renderer
https://stackoverflow.com/questions/42809368/angular-2-adding-a-class-to-a-dom-element-when-clicked
or even https://stackoverflow.com/a/47852581/1047852

**Pull request merged**